tests/test_qnode.py

```diff
@@ -25,7 +25,7 @@
 from defaults import pennylane as qml, BaseTest
 
 from pennylane.qnode import _flatten, unflatten, QNode, QuantumFunctionError
-from pennylane.plugins.default_qubit import CNOT, Rotx, Roty, Rotz, I, Y, Z
+from pennylane.plugins.default_qubit import CNOT, Rotx, Roty, Rotz, I, CRotx, CRoty, CRotz, Y, Z, X
```

Comment:
`X` doesn't appear to be used anywhere in this file, so no need to import it (unless you intended to make a test using it)